kartothek/io/dask/bag.py

```diff
@@ -239,7 +246,7 @@ def store_bag_as_dataset(
 
 @default_docs
 def build_dataset_indices__bag(
-    store, dataset_uuid, columns, partition_size=None, factory=None
+    store, dataset_uuid, columns, partition_size=None, factory=None, secondary_indices=None,
```

Comment:
This pipeline is intended for building indices and the argument `columns` covers already that use case